apps/condo/domains/property/components/PropertyAddressSearchInput.tsx

```diff
@@ -99,6 +99,7 @@ export const PropertyAddressSearchInput: React.FC<IAddressSearchInput> = (props)
             renderOption={renderOption}
             initialValueGetter={initialValueGetter}
             infinityScroll
+            searchAgainDependencies={[organizationId]}
```

Comment:
it's better to change the search function in case of changing the organization id

Reply:
remove it and memoized BaseSearchInput depends on `organization`